runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -501,6 +501,9 @@ TR::S390VirtualUnresolvedSnippet::emitSnippetBody()
 
    cursor = generatePICBinary(cg(), cursor, glueRef);
 
+   // J2I relocation information for private nestmate calls
+   auto j2iRelocInfo = reinterpret_cast<TR_RelocationRecordInformation*>(comp->trMemory()->allocateMemory(sizeof(TR_RelocationRecordInformation), heapAlloc));
```

Comment:
Can we please move this declaration as close as possible to where the data is actually initialized. No reason for it to live so far away making it's scope huge.

Reply:
Variable scope reduced.